spec/features/two_factor_authentication/sign_in_spec.rb

```diff
@@ -531,25 +531,6 @@ def attempt_to_bypass_2fa
 
     context 'sign in' do
       context 'with platform auth sign up enabled' do
```

Comment:
Can we get rid of this block altogether, since we don't have an opposing state anymore?